### PR TITLE
Updated EclipsePathAdapter to the latest Minimatch changes.

### PR DESCRIPTION
**Pull request by piotrtomiak**

### CODE REVIEW

File: eclipse/tern.eclipse.ide.core/src/tern/eclipse/ide/internal/core/scriptpath/EclipsePathAdapter.java [hunk 1]
```diff
@@ -10,10 +10,14 @@
  */
 package tern.eclipse.ide.internal.core.scriptpath;
```
Comment (angelozerr) [inline eclipse/tern.eclipse.ide.core/src/tern/eclipse/ide/internal/core/scriptpath/EclipsePathAdapter.java]: You have missed license header with your name

**Comment by piotrtomiak** [comment]
oh, well, I guess it's fine in this case ;) Nothing too fancy done here.

**Comment by angelozerr** [comment]
I think global pom must be changed to in order to use minimatch.java 1.1.0-SNAPSHOT https://github.com/angelozerr/tern.java/blob/master/pom.xml#L176

**Comment by piotrtomiak** [comment]
right... let me fix this

**Comment by piotrtomiak** [comment]
Looks like you will need to publish minimatch.java 1.1.0 to Maven repo

**Comment by angelozerr** [comment]
Use 1.1.0-SNAPSHOT

**Comment by piotrtomiak** [comment]
Build failed in a very weird way, can you kick it off again ?

**Comment by angelozerr** [comment]
I accept your PR. Let's me try

**Comment by paulvi** [comment]
@piotrtomiak Have you seen https://github.com/angelozerr/tern.java/issues/307 dependencies problem ?

**Comment by angelozerr** [comment]
@piotrtomiak tern.java build again.

@paulvi minimatch.java have none depdnencies. See https://github.com/angelozerr/minimatch.java/blob/master/META-INF/MANIFEST.MF I don't understand why there are this UI problem (perhaps it's because there are none depdnencies?)

**Pull request merged**